ts/components/PreferenceItem.tsx

```diff
@@ -1,11 +1,13 @@
-import { H3, Left, Right, Text } from "native-base";
+import { Left, Right, Text } from "native-base";
```

Comment:
Can't we replace this component with the new ListitemComponent?

Reply:
done in https://github.com/teamdigitale/io-app/pull/1175/commits/1821cb1a629d41ff828d7f0f671d4b38eb9e8b7d